bin/cron/cleanup_adhoc_environments

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+
+# Check if script is the only instance running
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../deployment'
+require 'aws-sdk-ec2'
+require 'aws-sdk-cloudformation'
+require 'cdo/chat_client'
+
+$dry_run = !rack_env?(:production)
+
+SECONDS_IN_A_DAY = 24 * 60 * 60
+
+def stop_running_adhoc_instances
+  adhocs = get_adhoc_instances('running')
```

Comment:
always a good practice if we had this as a const ('running')

or as a global enum

Reply:
Also this could be part of a class/library where you pass the "state".  I wouldn't print anything inside that method. 

I would make the method return any instances and then print the message

```
 msg = "Stopping #{adhocs.length} adhoc instances:\n" +
    adhocs.map {|a| a.tags.detect {|t| t.key == 'Name'}&.value}.join("\n")
```

Reply:
did some refactoring to stop passing strings around.